### ISSUE TITLE
Time Series Padding

### ISSUE THREAD

**Issue by khirotaka** [khirotaka/enchanter#15]
各サンプル毎で時系列の長さが違うと、 `DataLoader` で引っ張ってくる際に `torch.stack` 出来ずにエラーが起きる。

* [ ] `adjust_sequences(sequences, max_len=None, dtype=np.float32): `

を作成する。

**Comment by khirotaka** [comment]
`enchanter.engine.modules.pad_sequences`

**Comment by khirotaka** [comment]
https://github.com/khirotaka/enchanter/blob/94c4247912a45b5680a3591bb5b184bea12de3e6/enchanter/preprocessing/signal.py#L103

`max_len=None` で長さを指定するだけでなく、Numpy関数(min, mid, maxなど) を与えて None時の挙動を変えたい
